src/middlewares/requiredFieldsMiddleware.ts

```diff
@@ -38,7 +38,14 @@ const requiredFields = ({ getState, dispatch }: MiddlewareAPI<Dispatch<AnyAction
             state.view.widgets
             .filter(item => item.bcName === widget.bcName)
             .forEach(item => {
-                item.fields.forEach((widgetField: WidgetField) => {
+                let itemFieldsCalc: object[] = []
+                if (item.type === WidgetTypes.List) {
```

Comment:
There are more of editable (and hence validatable) widget types then `List` and `Form`, and there are more to come.  
 Client applications are also free to define their own widget types which also shouldn't be exempt here.